frontend/src/citizen-frontend/applications/editor/service-need/ServiceTimeSubSectionPreschool.tsx

```diff
@@ -108,55 +135,124 @@ export default React.memo(function ServiceTimeSubSectionPreschool({
         <>
           <Gap size="m" />
 
-          <ExpandingInfo
-            info={
-              t.applications.editor.serviceNeed.dailyTime.instructions[
-                applicationType
-              ]
-            }
-            ariaLabel={t.common.openExpandingInfo}
-            closeLabel={t.common.close}
-            inlineChildren
-          >
-            <Label>
-              {t.applications.editor.serviceNeed.dailyTime
-                .usualArrivalAndDeparture[applicationType] + ' *'}
-            </Label>
-          </ExpandingInfo>
+          {featureFlags.preschoolApplication.serviceNeedOption ? (
+            <>
+              <Label htmlFor={labelId}>
+                {t.applications.editor.serviceNeed.startDate.label['PRESCHOOL']}{' '}
+                *
+              </Label>
 
-          <Gap size="s" />
+              <Gap size="s" />
 
-          <FixedSpaceRow spacing="m">
-            <FixedSpaceColumn spacing="xs">
-              <Label htmlFor="daily-time-starts">
-                {t.applications.editor.serviceNeed.dailyTime.starts}
-              </Label>
-              <TimeInput
-                id="daily-time-starts"
-                value={formData.startTime}
-                data-qa="startTime-input"
-                onChange={(value) => updateFormData({ startTime: value })}
-                info={errorToInputInfo(errors.startTime, t.validationErrors)}
+              <DatePicker
+                date={formData.connectedDaycarePreferredStartDate}
+                onChange={(date) =>
+                  updateFormData({
+                    connectedDaycarePreferredStartDate: date
+                  })
+                }
+                locale={lang}
+                info={errorToInputInfo(
+                  errors.connectedDaycarePreferredStartDate,
+                  t.validationErrors
+                )}
                 hideErrorsBeforeTouched={!verificationRequested}
+                isInvalidDate={(date: LocalDate) =>
+                  isValidPreferredStartDate(
+                    date,
+                    originalPreferredStartDate,
+                    'PRESCHOOL',
+                    terms
+                  )
+                    ? null
+                    : t.validationErrors.unselectableDate
+                }
+                minDate={minDate}
+                maxDate={maxDate}
+                data-qa="connectedDaycarePreferredStartDate-input"
+                id={labelId}
+                required={true}
+                errorTexts={t.validationErrors}
```

Comment:
Could this (a separate starting date for preschool daycare) be something to move behind a separate feature flag? I don't think it's directly related to being able to select a service need.

Reply:
Yes, created a separate flag for this.